src/ios/CDVWKInAppBrowser.m

```diff
@@ -19,8 +19,8 @@ Licensed to the Apache Software Foundation (ASF) under one
 
 #import "CDVWKInAppBrowser.h"
 
-#if __has_include("CDVWKProcessPoolFactory.h")
-#import "CDVWKProcessPoolFactory.h"
+#if __has_include(<Cordova/CDVWebViewProcessPoolFactory.h>)
```

Comment:
In theory, for maximum compatibility, you could try both options:

```
#if __has_include(<Cordova/CDVWebViewProcessPoolFactory.h>) // Cordova-iOS >=6
  #import <Cordova/CDVWebViewProcessPoolFactory.h>
#elif __has_include("CDVWKProcessPoolFactory.h") // Cordova-iOS <6 with WKWebView plugin
  #import "CDVWKProcessPoolFactory.h"
#endif
```

Reply:
That makes sense since the requirements are just set to cordova-ios>5.1.0 in #823